### PR TITLE
geojson_to_array

### PR DESCRIPTION
**Pull request by tdefa**
function and script to save numpy array mask labels from the geojson files download from imjoy-cloud-annotation app

### CODE REVIEW

**Comment by muellerflorian** [comment]
Excellent. One suggestion. Currently the gejson is transformed into numpy, might be nice to actually save it as a label image (png). like this the workflow is completely closed and we re-create these images (which are often also the input from a segmentation approach).

**Comment by muellerflorian** [comment]
second comment. having a short mention in the docs would be nice.

File: docs/task-manager.md [hunk 1]
```diff
@@ -199,3 +199,8 @@ An example for a status file is shown below. Many of the fields are self-explana
     "completed_at": null
 }
 ```
+### Convert GeoJson to label images
+
+The uploaded dataset contains the annotations in GeoJson files and not in label images.
+
+We provide Python script (`utils\geojson_labels.py`), showing how to convert GeoJson files to label images for a nuclear segmentation. This code takes as input the path to the uploaded dataset and saves label images next to each GeoJson file.
```
Comment (oeway) [inline docs/task-manager.md]: Please use `/` in the path.

File: utils/geojson_labels.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import random
+from skimage.io import imread, imsave
+import numpy as np
+from pathlib import Path
+from utils_annotation import geojson_to_label
+import json
+
+# %%
+#path_results = Path(r'PASTE-PATH-TO-DATA')  # Folder with segmentation results from imjoy-cloud-annotation
+path_results = Path(r'//home/thomas/Bureau/phd/kaibu_data/test1')
```
Comment (oeway) [inline utils/geojson_labels.py]: Maybe replace this with the line above?

Comment (tdefa) [reply utils/geojson_labels.py]: ok

File: docs/task-manager.md (hunk 1 above)
Comment (tdefa) [reply docs/task-manager.md]: ok

**Pull request merged**